packages/rum-react/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@elastic/apm-rum-react",
+  "version": "0.0.0",
+  "description": "Elastic APM RUM React integration",
+  "author": "Hamid <h.jahtalab@gmail.com>",
+  "homepage": "https://www.elastic.co/guide/en/apm/agent/js-base/current/index.html",
+  "license": "MIT",
+  "main": "dist/lib/index.js",
+  "module": "src/index.js",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/elastic/apm-agent-rum-js.git"
+  },
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "script": "node ../../dev-utils/run-script.js",
+    "build": "webpack && npm run build:main",
```

Comment:
can we keep it consistent with apm-rum? 

```
"prepublishOnly": "npm run build && npm run build:main",
"build": "webpack",
```

Reply:
I think we should change the other ones to have only one `build` script.

Reply:
Sure,  I am fine with that change.